src/common/hosts-patches-table.js

```diff
@@ -89,11 +89,21 @@ class HostsPatchesTable extends Component {
     )
   )
 
-  _refreshMissingPatches = () => (
-    Promise.all(
-      map(this.props.hosts, this._refreshHostMissingPatches)
+  _subscribeMissingPatches = (hosts = this.props.hosts) => {
```

Comment:
Because I need to subscribe to the missing patches for each `this.props.hosts` and I can't access the props inside `utils/addSubscriptions`.